packages/redux/src/raffles/actions/__tests__/modifyRaffleParticipation.test.ts

```diff
@@ -0,0 +1,98 @@
+import * as actionTypes from '../../actionTypes';
+import { INITIAL_STATE } from '../../reducer';
+import { mockStore } from '../../../../tests';
+import { participationId_0, raffleId } from 'tests/__fixtures__/raffles';
+import {
+  patchRaffleParticipation,
+  RaffleParticipationStatus,
+} from '@farfetch/blackout-client';
+import find from 'lodash/find';
+import modifyRaffleParticipation from '../modifyRaffleParticipation';
+
+jest.mock('@farfetch/blackout-client', () => ({
+  ...jest.requireActual('@farfetch/blackout-client'),
+  patchRaffleParticipation: jest.fn(),
+}));
+
+const buildRaffleParticipationsMockStore = (state = {}) =>
+  mockStore({ result: INITIAL_STATE }, state);
+
+const expectedConfig = undefined;
+let store: ReturnType<typeof buildRaffleParticipationsMockStore>;
+
+describe('modifyRaffleParticipation() action creator', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    store = buildRaffleParticipationsMockStore();
+  });
+
+  const data = {
+    value: RaffleParticipationStatus.CANCELLED,
+    path: '/status',
+    op: 'replace',
+  };
+
+  it('should create the correct actions for when the modification raffle participation procedure fails', async () => {
+    const expectedError = new Error('participation error');
+
+    (patchRaffleParticipation as jest.Mock).mockRejectedValueOnce(
+      expectedError,
+    );
+    // expect.assertions(4);
```

Comment:
🙈

Reply:
removed

Reply:
Did you forgot to push it? It still there :D

Reply:
pls double check